vnext/src/Libraries/Color/normalizeColor.windows.js

```diff
@@ -0,0 +1,395 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Do you need to update the Copyright to Microsoft?

Reply:
We've been inconsistent about doing this across our forked files.  I believe it's fine for us to change the copyright notice, as the LICENSE being pointed to states that portions of the code are based on Facebook's implementation.  That's clearly the case here.  Will update and file an issue to update the others.

---
In reply to: [359086803](https://github.com/microsoft/react-native-windows/pull/3789#discussion_r359086803) [](ancestors = 359086803)